docs/jvs-quick-setup.md

```diff
@@ -72,11 +72,12 @@ will also get created.
 
 ### Justification API
 
-1.  Export the jvs server with the domain part of the `jvs_server_url` from
+1.  Export the jvs server and JWKS endpoint with the domain part of the `jvs_server_url` from
     Terraform outputs like `jvs-e2e-xxxx-uc.a.run.app`
 
     ```shell
     export SERVER=<jvs_server_domain>:443
+    export JWKS_ENDPOINT=<jvs_server_domain>:8080/.well-known/jwks
```

Comment:
@yolocs Could you confirm my understanding is correct here? thx!

Reply:
The jwks endpoint could be different from the JVS API server. But it's definitely preferable to put them behind the same GCLB and share a custom domain. I think we can do two things:

1. If the jwks endpoint is not specified in the config file, default the jwks endpoint to be `https://{SERVER}/.well-known/jwks`. We probably can't use viper for such defaulting. We need to handle that explicitly in code.

2. Allow overwriting the jwks endpoint via config/flag.

Reply:
I'm not sure if Viper supports it, but we could have JWS_ENDPOINT default to `${SERVER}:8080/.well-known/jwks`

Reply:
Ack.

Reply:
Minor correction. `SERVER` is the gRPC server address. It will be in the format of e.g. `example.com:443`. Without scheme `https://` and with port.
The JWT endpoint should be an http URL.

Reply:
To clarify, they should look like below?
```
SERVER=<jvs_server_domain>:443
Default JWKS_ENDPOINT=https://<jvs_server_domain>:8080/.well-known/jwks
```

Reply:
Updated, please take a look.